website/developer-docs/docs/templates/procedural-md.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Markdown template: procedural"
+---
+
+```
```

Comment:
I dont think escaping markdown within markdown is a good idea, rather this file should just be the template and we can link to github instead to view the raw template

Reply:
That does sound better... but how do I add a pure markdown file to GitHub without Netlify and Docusaurus converting it to html?

Reply:
You can just add it as is and let docusaurus do its thing and just not include it in the sidebar